projects/vdk-plugins/vdk-structlog/tests/test_structlog.py

```diff
@@ -56,6 +63,63 @@
 }
 
 
+def test_parse_log_level_module():
```

Comment:
It would be nice to have some tests for syslog output. If you think that's too involved for this PR, it can become a separate ticket.

Reply:
Yeah, I will include such tests as part of another PR.